Kernel/Arch/x86/ThreadRegisters.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2021, Gunnar Beutner <gbeutner@serenityos.org>
```

Comment:
@gunnarbeutner I added your copyright here, as a quick `git blame`, showed me that you initially came up with the ThreadRegisters concept. If that's not correct please tell me :)